src/commands/createNewProject/JavaScriptProjectCreator.ts

```diff
@@ -28,4 +32,33 @@ export class JavaScriptProjectCreator extends ScriptProjectCreatorBase {
             ]
         };
     }
+
+    public getTasksJson(runtime: string): {} {
+        let options: ITaskOptions | undefined;
+        if (runtime !== ProjectRuntime.one) {
+            options = {};
+            options.env = {};
+            options.env[funcNodeDebugEnvVar] = funcNodeDebugArgs;
+        }
+
+        return {
+            version: '2.0.0',
+            tasks: [
+                {
+                    label: funcHostTaskLabel,
+                    identifier: funcHostTaskId,
+                    type: 'shell',
+                    command: 'func host start',
+                    options: options,
```

Comment:
So this can return 

    options: undefined

is that okay?

Reply:
Yep. 'undefined' values are excluded when stringifying JSON, which is what we want in this case